source/jquery.flot.navigate.js

```diff
@@ -341,11 +341,9 @@ can set the default in the options.
             }
 
             if (o.pan.interactive) {
-                eventHolder.bind("dragstart", {
-                    distance: 10
-                }, onDragStart);
-                eventHolder.bind("drag", onDrag);
-                eventHolder.bind("dragend", onDragEnd);
+                plot.addEventHandler("dragstart", onDragStart, eventHolder, 0);
+                plot.addEventHandler("drag", onDrag, eventHolder, 0);
+                plot.addEventHandler("dragend", onDragEnd, eventHolder, 0);
                 eventHolder.bind("mousedown", onMouseDown);
                 eventHolder.bind("mouseup", onMouseUp);
```

Comment:
Is `addEventHandler` expressly limited to drag events?  Or are you just skipping these mouse events, because the extension this is for only happens to care about drag events?  Are drag events the only events that have multiple handlers?

Reply:
Skipping, because the extension (now) only cares about the drag events. There is no reason we couldn't just move to the new mechanism always, but for now it doesn't seem necessary.